README.md

```diff
@@ -1 +1,7 @@
-# common-collection
\ No newline at end of file
+# MyOnlineStore Common Collection
+
+## Install
+Install using composer
+```bash
+composer require myonlinestore/common-collection
```

Comment:
Perhaps it's worth noting that a custom Composer VCS root has to be defined in order to be able to install this library, since it's a private repository. Or would you assume that's 'common' knowledge? :smile:

Reply:
Shouldn't this package be made available through packagist.org? And thus not requiring a custom VCS root.

Reply:
I guess we can already add it to Packagist then. We can then test it without releases/tags using `dev-<branch>` notation.

Reply:
Packagist requires `composer.json` to be present in the `master`.
I suggest we merge this PR first, then add and test the package prior to tagging it.